### DISCUSSION TITLE
咋搞

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1693] [提问] [answered]
![image](https://user-images.githubusercontent.com/119654171/233758009-56db997a-1620-47db-be1e-abce8d59e7f7.png)

**Comment by WForst-Breeze** [comment, marked as answer]
感觉是用的虚拟机吧，手机上能直接使用的 Java 版启动器我知道的应该就是 HMCLPE 和 PojavLauncher，PCL2 目前不支持直接在其他操作系统上直接运行，更何况在移动设备上。手机虚拟机可以尝试使用 lbochs，您可以自己去相关网站搜索详细教程，并在安装好的虚拟机上运行 PCL2。还有部分电脑模拟器里运行也是有可能的……

edit: 参考下方讨论，就是用模拟器运行的

**Reply by iPodtouch6557** [reply]
不不不不不，就是我教的，结果装错地了
能用只是他们俩和我说的，我不确定，我也没见过

**Reply by Chiloven945** [reply]
> 不不不不不，就是我教的，结果装错地了 能用只是他们俩和我说的，我不确定，我也没见过

所以说他俩是怎么在手机上运行exe的（

**Reply by iPodtouch6557** [reply]
> > 不不不不不，就是我教的，结果装错地了 能用只是他们俩和我说的，我不确定，我也没见过
> 
> 所以说他俩是怎么在手机上运行exe的（

应该……在exe下载到安卓上面时会改成api文件？同理，安卓在下到电脑上时也会改api文件，这我确定，~之前下载模拟地铁时就由于没分清手机页面和电脑页面下了个api文件，结果用联想模拟器打开了~

**Reply by so1ve** [reply]
> > > 不不不不不，就是我教的，结果装错地了 能用只是他们俩和我说的，我不确定，我也没见过
> > 
> > 
> > 所以说他俩是怎么在手机上运行exe的（
> 
> 应该……在exe下载到安卓上面时会改成api文件？同理，安卓在下到电脑上时也会改api文件，这我确定，~之前下载模拟地铁时就由于没分清手机页面和电脑页面下了个api文件，结果用联想模拟器打开了~

首先是答案，用模拟器开的

> 之前下载模拟地铁时就由于没分清手机页面和电脑页面下了个api文件，结果用联想模拟器打开了

错误的，这只是联想模拟器配置了APK的默认打开方式，所以让你双击能打开它，它本身还是APK不是一个EXE

再说了，APK和EXE这俩东西完全不一样你的系统怎么帮你转来转去（）

**Reply by iPodtouch6557** [reply]
不是啊……本来也没说转来转去，我就是意外发现只要是apk就能用联想模拟器打开
